openpype/hosts/maya/api/lib.py

```diff
@@ -16,6 +16,15 @@
 
 from maya import cmds, mel
 import maya.api.OpenMaya as om
+from arnold import (
+    AiTextureGetBitDepth,
+    AiTextureGetFormat,
+    AiTextureInvalidate,
+    # types
+    AI_TYPE_BYTE,
+    AI_TYPE_INT,
+    AI_TYPE_UINT
+)
```

Comment:
Oh boy, we shouldn't be doing this! :) Arnold isn't a hard requirement to be installed with maya.